### PR TITLE
Added known issue and updated anchorectl num

### PR DESCRIPTION
**Pull request by anchoredeanlappi**
Added known issue information to 3.3.0 release notes.
Updated Anchorectl numbering.

Signed-off-by: anchoredeanlappi <88210304+anchoredeanlappi@users.noreply.github.com>

### CODE REVIEW

File: content/docs/releasenotes/330.md [hunk 1]
```diff
@@ -23,6 +23,136 @@ Anchore Enterprise can now scan Rocky Linux images for vulnerabilities.
...
 
+### Known Issues
+
+NVD CVSS scores are missing in the query vulnerabilities API with the Grype provider.
+
+#### Problem 
+The `/query/vulnerabilities` API response contains an `nvd_data` attribute for each vulnerability in the result. The value of the attribute represents the NVD assigned CVSS scores. This field is not correctly populating for a small subset of vulnerabilities in the system. Instead of a list of results, the value is a null reference as shown below. 
+**Note:** This known issue only affects vulnerabilities that exclusively belong in the `nvd` namespace with Grype as the vulnerabilities provider (Next-gen v2 scanner). It does not affect the legacy vulnerability provider.
+
+    % curl -u user:password "http://localhost:8228/v1/query/vulnerabilities?id=CVE-2019-15780"
```
Comment (nightfurys) [inline content/docs/releasenotes/330.md:34]: Can this be formatted as a code snippet? Has to be enclosed within ```

File: content/docs/releasenotes/330.md [hunk 2]
```diff
@@ -23,6 +23,136 @@ Anchore Enterprise can now scan Rocky Linux images for vulnerabilities.
...
+          "vendor_data": []
+        }
+      ]
+    }
+
+#### Workaround
+
+The API supports a namespace query parameter to filter results based on the namespace. Supply the namespace with an `nvd` value to view the NVD CVSS scores, as shown in the following example.
+
+    % curl -u user:password "http://localhost:8228/v1/query/vulnerabilities?id=CVE-2019-15780&namespace=nvd"
```
Comment (nightfurys) [inline content/docs/releasenotes/330.md:82]: Can this be formatted as a code snippet? Has to be enclosed within ```

**Review by nightfurys — CHANGES_REQUESTED** [review]
Looks good overall, requesting a formatting change for code snippets

File: content/docs/releasenotes/330.md (hunk 1 above)
Comment (nightfurys) [reply content/docs/releasenotes/330.md:34]: @anchoredeanlappi looks like spaces have a similar effect, retracting my request here

File: content/docs/releasenotes/330.md (hunk 2 above)
Comment (nightfurys) [reply content/docs/releasenotes/330.md:82]: @anchoredeanlappi looks like spaces have a similar effect, retracting my request here

**Comment by nightfurys** [comment]
Nice work on the change log! Can you also update _index.md file and add all the missing links?

File: content/docs/releasenotes/330.md [hunk 3]
```diff
@@ -23,6 +23,140 @@ Anchore Enterprise can now scan Rocky Linux images for vulnerabilities.
 - With the Grype provider, NVD and vendor CVSS scores were missing for records in non-NVD namespaces. This is now fixed.
 - Migration code was added to clean-up the unused feed records, and fixed artifacts and vulnerabilities records for the github:os group.
 
+### Known Issues
+
+NVD CVSS scores are missing in the query vulnerabilities API with the Grype provider.
```
Comment (zhill) [inline content/docs/releasenotes/330.md:28]: I'd like to be much more specific about this and indicate there is a workaround here, so folks don't have to read down the whole item to see that.
```suggestion
NVD CVSS scores may not be present in the API responses for the request to get detailed information query about a vulnerability feed record. 
* There is a workaround to get this information, see below for more detail on the workaround.
* This is only present for a subset of records NVD records.
* It does not impact the vulnerability reports or findings for images. It only impacts the next-gen vulnerability scanner so users still on the legacy scanner are not impacted.
```

File: content/docs/releasenotes/330.md [hunk 4]
```diff
@@ -23,6 +23,140 @@ Anchore Enterprise can now scan Rocky Linux images for vulnerabilities.
...
+
+#### Problem
```
Comment (zhill) [inline content/docs/releasenotes/330.md:30]:
```suggestion
#### Detail
```

**Review by zhill — CHANGES_REQUESTED** [review]
Suggestions on the wording to make the initial summary more clear to users so they don't have to read the whole detail to understand if it impacts them or not.

File: content/docs/releasenotes/330.md (hunk 3 above)
Comment (anchoredeanlappi) [reply content/docs/releasenotes/330.md:28]: Good point. I will add it

**Review by zhill — APPROVED** [review]
lgtm!

**Pull request merged**